kolibri/core/assets/src/core-app/seededshuffle.js

```diff
@@ -0,0 +1 @@
+module.exports = require('seededshuffle');
```

Comment:
Maybe lets get https://github.com/learningequality/kolibri/pull/1324 merged, and remove these sooner rather than later!